tests/tools/onert_run/src/rawformatter.h

```diff
@@ -28,13 +27,13 @@ struct nnfw_session;
 
 namespace onert_run
 {
-class RawFormatter : public Formatter
+class RawFormatter
```

Comment:
Is there any possibility that `RawFormatter` is used through `Formatter` interface?

Reply:
I can't find that.